src/main/scala/com.snowplowanalytics.sauna/SaunaOptions.scala

```diff
@@ -145,7 +147,7 @@ object SaunaOptions {
   def buildConfigMap(files: List[File]): Either[String, Map[String, List[Envelope]]] = {
     val enabledConfigs = sequence(files.map(parseSelfDescribing)).right.map { configs =>
       configs.filter(filterEnabled).groupBy(_.schema.name)
-    }.left.map( list => list.mkString(", "))
+    }.left.map(list => list.mkString(", "))
```

Comment:
This is minor, but generally discouraged to put these changes in commit log - they will obscure git blame.